client/app/src/accounts/account.controller.js

```diff
@@ -586,12 +586,30 @@
       }
 
       var currencyBalance = Number(balance * price).toLocaleString(languageCode, options);
-      
+
       if (currencyName == "btc") currencyBalance = currencyBalance.replace("BTC", "Ƀ");
 
       return currencyBalance;
     }
 
+    self.formatCurrencyVal = function(val) {
```

Comment:
`formatCurrencyBalance` and `formatCurrencyVal` sounds very similar, can you merge both into a single function?